answers/final.py

```diff
@@ -60,9 +59,7 @@ def get_answer(question, graph, final=False):
 
     grapheize(q_graph, q_doc, attrs={'q_type': q_type})
     scores = Counter()
-    found_common = False
     for start, end in q_graph.edges():
-        found_common = True
```

Comment:
@Man-Jain  don't make changes to `answers/*` and push your commit again. I am not sure, but this might work.

Reply:
^Ignore